src/pages/Offers/Offers.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import {
+  IonButton,
+  IonButtons,
+  IonHeader,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+} from "@ionic/react";
+import useAuth from "hooks/useAuth/useAuth";
+
+const Offers = () => {
+  const { logout } = useAuth();
+
+  return (
+    <IonPage>
+      <IonHeader>
```

Comment:
I want to create a Header component for this since the exact header code is also repeated in Home.

If I create the Header component and write its unit tests e.g. renders log out button, clicking it takes me to /login, etc. does that mean I can remove those same tests from Home and Offers?